### PR TITLE
Migrate to Django

### PR DESCRIPTION
**Pull request by zhangjiangqige**
Important changes:

- run by `python manage.py runserver`, as in Procfile
- because manage.py doesn't accept arguments, `config.json` is hard coded at config.py:42. When developing, let's just overwrite/remove-and-soft-link config.json for now

### CODE REVIEW

**Comment by zhangjiangqige** [comment]
When developing a new interface (such as house for /house/create etc.):

1. run `python manage.py startapp house`, and a new folder named "house" will be created
2. go to soy_sauce/urls.py and add an global url pattern, similar to Blueprint in Flask
3. go to house/urls.py and add url patterns for APIs under "house"
4. define a db model in house/models.py (see user/models.py as an example)
5. run `python manage.py makemigrations house` to somehow process the model
6. run `python manage.py migrate` to create a collection in MongoDB (must run)
7. write view functions in house/views.py

File: soy_sauce/settings.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'djongo',
+        'ENFORCE_SCHEMA': True,
+        'NAME': conf['database_name'],
```
Comment (xingweitian) [inline soy_sauce/settings.py:124]: Do we need to use "database_name"? I think it is included in "database_uri".

Comment (xingweitian) [reply soy_sauce/settings.py:124]: Or, do we need to remove "birana" from the end of the `database_uri`?

Comment (zhangjiangqige) [reply soy_sauce/settings.py:124]: Both are needed by djongo. This is also required if we were to use plain pymongo.

File: website/urls.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    path('ping', views.ping, name='ping'),
+]
```
Comment (xingweitian) [inline website/urls.py]: A new line is needed here.

File: website/views.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django.http import HttpResponse
+
+# Create your views here.
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")
+
+def ping(request):
+    return HttpResponse('pong')
```
Comment (xingweitian) [inline website/views.py]: A new line is needed here.

**Review by xingweitian — APPROVED** [review]
Just add two new lines and then I think there are no more problems here.

**Pull request merged**